app/src/main/java/com/rob729/quiethours/Activity/SplashScreen.kt

```diff
@@ -16,6 +16,7 @@ class SplashScreen : AppCompatActivity() {
         startActivity(intent)
         overridePendingTransition(R.anim.fade_in, R.anim.fade_out)
         checkFirstRun()
+        checkAppIntro()
```

Comment:
There is already a startActivity() present above checkFirstRun() and then you are adding similar code in checkAppIntro().
Make the changes for the proper working of the app.